app/src/main/java/com/jerboa/ui/components/common/MarkdownHelper.kt

```diff
@@ -150,13 +152,33 @@ object MarkdownHelper {
                 }
             })
             .build()
+
+        noImageMarkwon = Markwon.builder(context)
```

Comment:
These are mostly identical, so you could probably use a common builder. Then at the very end, create the two versions:

```
imageVersion = builder.usePlugin(coil...).build()
nonImageVersion = builder.build()
```